### PR TITLE
 [support] Update polkadot dependencies

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

Update `@polkadot/*` dependencies

### ❓ Context

- **Impacted projects**: `LLC` `LLD` `LLM`
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-8984

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

Screenshot 2023-11-21 at 14 59 19

Simulator Screenshot - iPhone 13 - 2023-11-21 at 15 08 50

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by haammar-ledger** [comment]
(replacing https://github.com/LedgerHQ/ledger-live/pull/4739 to attempt to pass CI)

**Comment by gre** [comment]
tip for next time if you reach timeout issue on CI = they usually mean the app never loads / fails to load.

you can download the results from the CI

Screenshot 2023-10-19 at 17 17 08


you extract one and you can check both the html report and the videos

Screenshot 2023-10-19 at 17 17 24

Screenshot 2023-10-19 at 17 17 51

from the video we see the problem

Screenshot 2023-10-19 at 17 17 57

**Comment by haammar-ledger** [comment]
@lvndry I know that you tested builds before, and I still don't see a link between the error and the changes... please see if you reproduce by testing builds now, but there's something weird here.

**Review by haammar-ledger — APPROVED** [review]

**Review by gre — CHANGES_REQUESTED** [review]
mmh, our goal should be to deduplicate polkadot , not add more dup. 
i'm looking at the typo with the `in and` in that automatic message, but we need to look why we didn't solve the polkadot dup yet 🤔

**Review by gre — APPROVED** [review]
looks like the bundle size script is showing improvement & things finally deduplicated 🚀

File: apps/ledger-live-desktop/src/renderer/families/polkadot/Nomination/Row.tsx [hunk 1]
```diff
@@ -109,16 +117,19 @@ export function Row({
...
     [unit, total],
   );
+
   const formattedCommission = useMemo(
     () => (commission ? `${commission.multipliedBy(100).toFixed(2)} %` : "-"),
     [commission],
   );
+
   const onExternalLinkClick = useCallback(() => onExternalLink(address), [onExternalLink, address]);
+  console.log({ validator });
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/Nomination/Row.tsx:127]: `console.log` to remove :eyes:

**Review by haammar-ledger — APPROVED** [review]
@lvndry For changes with visual impact you can provide before/after screenshots, to illustrate the change and prove that it was tested :+1:

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**